forms/src/main/java/org/devgateway/toolkit/forms/application.properties

```diff
@@ -10,4 +10,10 @@
 # Development Gateway - initial API and implementation
 ###############################################################################
 spring.profiles.active=reports
-server.port = 8090
\ No newline at end of file
+server.port = 8090
+
+# Defines role hierarchy.
+# Hierarchy is specified as a string. Space separates rules and > symbol has the meaning of 'includes'.
+# Example: role1 > role2 > role3 role2 > role4
+# Here role1 includes role2 role3 and role4 (indirectly). And role2 includes role4.
+roleHierarchy=ROLE_ADMIN > ROLE_USER
```

Comment:
is this something that sits better in the forms module or in the web module ? the web module still supports spring security but is meant to be used just with spring mvc, and/or endpoints, and no wicket. if you think the hierarchical roles can be useful there too, i recommend we move this to toolkit/web/application.properties.